api.rst

```diff
@@ -493,6 +528,36 @@ Calling the function returns HTTP 400 with the body
 
 You can customize the HTTP status code by raising particular exceptions according to the PostgREST :ref:`error to status code mapping <status_codes>`. For example, :code:`RAISE insufficient_privilege` will respond with HTTP 401/403 as appropriate.
 
+Raising Errors
```

Comment:
Did this section slip in by mistake during the rebase? It exists already at https://postgrest.com/en/v4.1/api.html#raising-errors